crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -920,15 +920,34 @@ pub enum AuthorizedotnetWebhookEvent {
     RefundCreated,
 }
 
-impl From<AuthorizedotnetWebhookEvent> for api::IncomingWebhookEvent {
-    fn from(event_type: AuthorizedotnetWebhookEvent) -> Self {
+#[derive(Debug, Deserialize)]
+pub enum AuthorizedotnetIncomingWebhookEventType {
```

Comment:
How is this different from `AuthorizedotnetWebhookEvent`? Is there a need for duplicating the enum?

Reply:
Yes , there is , as so earlier `AuthorizedotnetWebhookEvent` enum was being implemented as a conversion From,   for both `SyncStatus` enum as well as `IncomingWebhookEvents` enum so , if we use the same enum there would be a need for matching the `AuthorizedotnetWebhookEvent` enum's `Unknown` field with one of `SyncStatus` enum's field.

Reply:
Fair enough, would you mind documenting the need for the new enum as comments in code?